geniza/corpus/templates/admin/corpus/document/merge.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'admin/base_site.html' %}
+{% block title %} Merge selected documents {% endblock %}
+
+{% block breadcrumbs %}
+    {% if not is_popup %}
+        <ul>
+            <li><a href="{% url 'admin:index' %}">Home</a></li>
+            <li><a href="{% url 'admin:app_list' app_label='corpus' %}">Corpus</a></li>
+            <li><a href="{% url 'admin:corpus_document_changelist'%}">Document</a></li>
+            <li>Merge selected documents</li>
+        </ul>
+    {% endif %}
+{% endblock %}
```

Comment:
These breadcrumbs don't match the styles for the rest of the admin site; is this still based on the mep/grappelli version?

See if there's a base template that's closer to what we need, or else adjust to be closer to the format django uses for displaying breadcrumbs.